### PR TITLE
chore: ACs added to 6002-MDET-market-details

### PR DESCRIPTION
**Pull request by radoslaw-szpiech**
Relates to https://github.com/vegaprotocol/frontend-monorepo/issues/3505

### CODE REVIEW

**Review by bwallacee — COMMENTED** [review]
Looks good.

Only thing I can think of, do we want ACs to cover the hover tooltips at all?

File: user-interface/6002-MDET-market-details.md [hunk 1]
```diff
@@ -1 +1,118 @@
-# Market details
\ No newline at end of file
+# Market 
+
+As a trading platform user I want to see all possible information about market.  
+<i>IMPORTANT: Available entries may vary depending on market specifics (e.g. oracle related). Separate ACs may be created for more detailed information.</i>
+
+## Market details
+
+When I am on trading platform I **Must** see following market details summary:
```
Comment (daro-maj) [inline user-interface/6002-MDET-market-details.md]: When I am on trading platform I **Must** see following market details summary:
```suggestion
When I am on trading platform I **Must** see following market details summary on the top of trading page:
```

File: user-interface/6002-MDET-market-details.md [hunk 2]
```diff
@@ -1 +1,118 @@
...
+
+- Market name (<a name="6002-MDET-001" href="#6002-MDET-001">6002-MDET-001</a>)
+- Expiry (<a name="6002-MDET-002" href="#6002-MDET-002">6002-MDET-002</a>)
```
Comment (daro-maj) [inline user-interface/6002-MDET-market-details.md:11]: For all those fields - what about information if tooltip should be  visible ?

**Review by daro-maj — DISMISSED** [review]

**Comment by radoslaw-szpiech** [comment]
> 

@ben-vegaprotocol @daro-maj  in my understanding (and also that was a conclusion in latest conversation in our team) that would be goal of separate ACs (e.g. Trading mode tooltip content can vary depending on... well... trading mode ;) and putting all possible options into this one is not a great idea. But it does not mean we cannot add some tests for that even without ACs.

**Comment by bwallacee** [comment]
@radoslaw-szpiech that makes sense.

Would a middle ground approach be to have an AC that tooltips can be displayed. Then we have one test to ensure in this panel a tooltip can be displayed? (not asserting the text itself, just making sure a tooltip and some text is displayed?)

**Review by bwallacee — DISMISSED** [review]

File: user-interface/6002-MDET-market-details.md [hunk 3]
```diff
@@ -1 +1,118 @@
...
\ No newline at end of file
+# Market 
+
+As a trading platform user I want to see all possible information about market.  
+<i>IMPORTANT: Available entries may vary depending on market specifics (e.g. oracle related). Separate ACs may be created for more detailed information.</i>
+
+## Market details
+
+When I am on trading platform I **Must** see following market details summary on the top of trading page:
+<i> Depending on market data tooltips may be available in some records when hovered.</i>
```
Comment (bwallacee) [inline user-interface/6002-MDET-market-details.md]:
```suggestion
<i> Depending on the market data, tooltips may be available for some records when hovered on.</i>
```

**Review by bwallacee — APPROVED** [review]

**Pull request merged**